spec/models/user_spec.rb

```diff
@@ -0,0 +1,53 @@
+require "rails_helper"
+
+RSpec.describe User, type: :model do
+  let!(:section) {FactoryBot.create :section}
```

Comment:
let! và let khác gì nhau

Reply:
`let` is lazy-evaluated: it is not evaluated until the first time the method it defines is invoked.
Using `let!` to force the method's invocation before each example. It means even if you didn't invoke the helper method inside the example, still it will be invoked before your example runs.